api/src/main/java/com/okta/idx/sdk/api/client/IDXAuthenticationWrapper.java

```diff
@@ -586,4 +591,51 @@ private AuthenticationTransaction selectPasswordAuthenticatorIfNeeded(Authentica
     public boolean isSkipAuthenticatorPresent(ProceedContext proceedContext) {
         return proceedContext.getSkipHref() != null;
     }
+
+    public AuthenticationResponse getIdps() {
+
+        List<Idp> idpList = new LinkedList<>();
+        AuthenticationResponse authenticationResponse = new AuthenticationResponse();
+
+        try {
+            AuthenticationTransaction introspectTransaction = AuthenticationTransaction.create(client);
+            authenticationResponse.setProceedContext(introspectTransaction.createProceedContext());
+
+            RemediationOption[] remediationOptions = introspectTransaction.getResponse().remediation().remediationOptions();
+
+            List<RemediationOption> remediationOptionList = Arrays.stream(remediationOptions)
+                    .collect(Collectors.toList());
+
+            for (RemediationOption remediationOption : remediationOptionList) {
```

Comment:
2 things:
1. Should we add a filter here, so we only get remediation options with idps?
2. Should we move this logic into the `AuthenticationTransaction#asAuthenticationResponse`, that way any step in the authentication flow can use idps?

Reply:
done